README.md

```diff
@@ -1,32 +1,90 @@
 # Arizona Icons
+Find more detailed information on how to use `az-icons` within the (Arizona
+Bootstrap Docs)[https://digital.arizona.edu/arizona-bootstrap/docs/2.0/icons/].
+## Download
+Download Arizona Icons to get the compiled CSS and font files, source code, or
+include it with your favorite package managers like npm, RubyGems, and more.
 
-## Using the Icons
-The icons can be used with Arizona Digital CDN link:
+### Source Files
+Compile Arizona Icons with your own asset pipeline by downloading our source
+CSS, SVG, PNG, and font files. This option requires some additional tooling:
 
-```html
-<link href="https://cdn.digital.arizona.edu/lib/az-icons/main/az-icons-styles.css" rel="stylesheet">
+### Arizona Digital CDN
+Skip the download with the Arizona Digital CDN to deliver cached version of
+Arizona Icons’s CSS and font files to your project.
+
+When choosing which version to include in your project, take these three choices
+into account:
+
+**Stable** version: This option is the most recent version that has the most
+wide-spread adoption on sites.
+```
+<link rel="stylesheet" href="https://cdn.digital.arizona.edu/lib/az-icons/1.0.0/az-icons-styles.css" crossorigin="anonymous">
+```
+
+**Latest** version: Latest tagged version. The most recently tagged stable
```

Comment:
Could the use of "stable" here be confusing when comparing to the "stable version"?

Reply:
I'm not sure, what about:

```
Latest tagged version. This contents of this CDN link update automatically, every time we tag a release of `az-icons`. While there is some risk that if you use this link, the icons could randomly break on your site, the up-side is that you get access to new icons and upgrades to performance on each release.
```